src/app/communities/nft/[address]/page.tsx

```diff
@@ -272,6 +277,12 @@ export default function Page() {
             <div className="font-semibold">{nftInfo?.title}</div>
           </div>
           <div className="mb-[24px] font-extralight">{nftInfo?.longDescription}</div>
+          {hasActiveCampaign && isBoosted && (
```

Comment:
This appears in all the cards, and it should be only in the active one.

Reply:
adding condition